taskipy/exceptions.py

```diff
@@ -0,0 +1,66 @@
+class TaskipyError(Exception):
```

Comment:
👍 in the `further improvements` section, I had noticed this may also get out of hand quite quickly. But this solution is much cleaner than the idea I had presented there